modules/medical/index.js

```diff
@@ -63,10 +63,10 @@ class AlmondUser extends BaseAlmondUser {
         if (rule.query.isStar || rule.query.isClassStar)
             rule.query = new TT.Ast.PermissionFunction.Specified('com.mai-hub', 'find', TT.Ast.BooleanExpression.True, null);
 
-        rule.query.filter = TT.Ast.BooleanExpression.And([
-            rule.query.filter,
-            TT.Ast.BooleanExpression.Atom('patient', '==', TT.Ast.Value.String(this._dbUser.cloud_id))
-        ]).optimize();
+        // rule.query.filter = TT.Ast.BooleanExpression.And([
+        //     rule.query.filter,
+        //     TT.Ast.BooleanExpression.Atom('patient', '==', TT.Ast.Value.String(this._dbUser.cloud_id))
+        // ]).optimize();
```

Comment:
This change will allow patients to read everybody's data. Which is ok for a controlled demo but is obviously wrong in general, and something to keep in mind.